Fix incorrect virtual network key during import of virtual_network_address_range

philipfreude opened the pull request:
Fixes #456

<!--- Please keep this note for the community --->

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for PR followers and do not help prioritize the request

<!--- Thank you for keeping this note for the community --->

### Description

<!--- Please leave a helpful description of the PR here. --->

Fix incorrect virtual network key during import of virtual_network_address_range

### References

Close #456

### New or Affected Resource(s)

<!--- Please list the new or affected resources and data sources. --->

- opennebula_virtual_network

### Checklist

<!--- Please check you didn't forgot a step to help us merging this PR. --->

- [x] I have created an issue and I have mentioned it in `References`
- [x] My code follows the style guidelines of this project (use `go fmt`)
- [x] My changes generate no new warnings or errors
- [ ] I have updated the unit tests and they pass succesfuly
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation (if needed)
- [x] I have updated the changelog file

philipfreude committed:
Fix incorrect virtual network key during import of virtual_network_address_range

Fixes #456

treywelsh commented:
Thank you for this PR :)

It would be nice to add something like this to the CHANGELOG file:
```
# 1.3.0 (Unreleased)

BUG FIXES:

* resources/opennebula_virtual_network: fix virtual network ID key in import method
```

philipfreude committed:
Fix incorrect virtual network key during import of virtual_network_address_range - add change to CHANGELOG.md

Fixes #456

philipfreude commented:
Sorry about that! I updated the `CHANGELOG.md`.

I don't think this would justify a minor version bump, so I just updated the patch version. Is that alright with you?

frousselet reviewed (approved):
Many thanks! 🙂

frousselet committed:
Merge branch 'master' into fix-opennebula_virtual_network_address_range-import
